603-spring-web-smallrye-openapi/src/test/java/org/acme/spring/web/OpenApiTest.java

```diff
@@ -8,40 +8,34 @@
 
 import org.apache.http.HttpStatus;
 import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.MethodOrderer;
-import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.TestMethodOrder;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvFileSource;
 
 import io.quarkus.test.junit.QuarkusTest;
 import io.restassured.response.Response;
 
 @QuarkusTest
-@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 public class OpenApiTest {
-    private static Response response;
+    private Response response;
 
-    private void callOpenApiEndpoint() {
+    @BeforeEach
```

Comment:
Thanks for sorting this out. The original intention was to save a few requests, but the independence of the tests is better. ;)